internal/plugin/plugin.go

```diff
@@ -230,7 +231,9 @@ func FilterByChangedFiles(plugins []*Plugin, lookuper envconfig.Lookuper) ([]*Pl
 
 func ParsePluginsEnvVar(pluginsEnv string) ([]IncludePlugin, error) {
 	var includes []IncludePlugin
-	fields := strings.Fields(pluginsEnv)
+	fields := strings.FieldsFunc(pluginsEnv, func(r rune) bool {
+		return unicode.IsSpace(r) || r == ','
```

Comment:
This has tripped me up a few times and space delimited values is a pain to work with sometimes in the shell so also accept comma delimited plugins.